### ISSUE TITLE
Markdown template for report in a package

### ISSUE THREAD

**Issue by dirkschumacher** [R4EPI/projmgmt#7]
Might be a nice idea to have an RStudio markdown template as part of a package

**Comment by zkamvar** [comment]
Similar to the template that @epiamsterdam provided last week with pre-prepared sections such as medical/epi information, Assessment, Objectives, and Progress?

**Comment by dirkschumacher** [comment]
Yeah something like that. And then maybe some code snippets that you can use, adapt or delete
